### PR TITLE
Preview workflow

### PR DESCRIPTION
**Pull request by a-belhadj**

### CODE REVIEW

File: profiles/views/organization.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
 from Squest.utils.squest_table import SquestRequestConfig
 
 from Squest.utils.squest_views import *
+from profiles.api.serializers import scope_serializer
```
Comment (EliasBoulharts) [inline profiles/views/organization.py:4]: to remove

File: service_catalog/api/urls.py [hunk 2]
```diff
@@ -87,7 +87,6 @@
     # ApprovalWorkflow update postion of steps
     path('approval-workflow/<int:pk>/update-steps-position/', ApprovalWorkflowUpdateStepsPosition.as_view(),
          name='api_approvalworkflow_update_steps_position'),
-
```
Comment (EliasBoulharts) [inline service_catalog/api/urls.py:90]: why remove this space?

File: profiles/tables/approval_workflow.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    name = LinkColumn()
+    preview = TemplateColumn(template_name='profiles/custom_columns/preview_workflow.html', orderable=False)
+
+    class Meta:
+        model = Scope
+        attrs = {"id": "role_table", "class": "table squest-pagination-tables"}
+        fields = ("name", "preview")
+
+    def render_name(self, value, record):
+        return format_html(f'<a title={record} href="{record.get_absolute_url()}">{record}</a>')
```
Comment (EliasBoulharts) [inline profiles/tables/approval_workflow.py:19]: duplicate already present in service_catalog

File: templates/profiles/custom_columns/preview_workflow.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<a href="{% url "service_catalog:approvalworkflow_preview" object.id record.id %}">
+    Preview
+</a>
```
Comment (EliasBoulharts) [inline templates/profiles/custom_columns/preview_workflow.html:3]: To remove (duplicate)

File: service_catalog/models/operations.py [hunk 5]
```diff
@@ -52,10 +52,14 @@ class Operation(SquestModel):
                                  help_text="Jinja supported. Job template type")
     validators = CharField(null=True, blank=True, max_length=200, verbose_name="Survey validators")
 
+    @property
+    def validators_name(self):
+        return self.validators.split(",")
```
Comment (EliasBoulharts) [inline service_catalog/models/operations.py:57]: if self.validator else None

File: service_catalog/views/doc.py [hunk 6]
```diff
@@ -20,6 +20,9 @@ class DocListView(SquestListView):
     model = Doc
     filterset_class = DocFilter
 
+    def get_queryset(self):
+        return Doc.objects.filter(services__isnull=True, operations__isnull=True)
+
```
Comment (EliasBoulharts) [inline service_catalog/views/doc.py:25]: Test must be update

**Review by EliasBoulharts — APPROVED** [review]

**Review by Sispheor — APPROVED** [review]

**Pull request merged**